cmd/proxy-server/main.go

```diff
@@ -52,13 +52,10 @@ import (
 )
 
 const (
-	certificateCrtFile = "certificate.crtfile"
-	certificateKeyFile = "certificate.keyfile"
-	rootCertFile       = "certificate.rootcertificate"
-	sidecarProxyAddr   = "web.sidecarproxyaddr"
-	jwtSigningScrt     = "web.jwtsigningsecret"
-	logLevel           = "LOG_LEVEL"
-	logFormat          = "LOG_FORMAT"
+	configParamSidecarProxyAddr = "web.sidecarproxyaddr"
```

Comment:
Would it be a lot of work to have a parameter for sidecar-image tar location? Right now we always store it in the root dir ("/")

Reply:
The sidecar image gets built in the working directory. As discussed, we'll keep it as is.